Assets/Scripts/Models/Buildable/Utility.cs

```diff
@@ -80,6 +84,7 @@ private Utility(Utility other)
         typeTags = new HashSet<string>(other.typeTags);
         description = other.description;
         Tint = other.Tint;
+        deconstructInventory = other.deconstructInventory;
```

Comment:
It actually doesn't matter. This inventory is copied later on. I can't imagine why one particular instance of a type of furniture would need to have different inventory requirements than any other would. As a result, it saves quite a bit of memory to just have a pointer to the proto object for this list, and not actually copy the list.